stylesheets/scss/layouts/_gui.scss

```diff
@@ -46,11 +46,28 @@
         border-radius: $gui-component-radius;
         box-shadow: $box-shadow-frame;
         background-color: $white;
-        border-left: 4px solid #A4ADFB;
         padding: .5rem;
         margin-bottom: .5rem;
         @include user-select-none;
         cursor: move;
+
+        // @PF: If colors are not wanted for the bus components on the sidebar (just one generic bus), 
+        //      remove the following classes
+        &--bus-electricity {
```

Comment:
the drawflow.min.css  for `.drawflow .drawflow-node .input` is breaking you design, how to overwrite it?

Reply:
I've checked the CSS on the platform and updated it here too. Does it work better now?

Reply:
Yes it works better :)